packages/suite-desktop/scripts/build.js

```diff
@@ -58,7 +58,8 @@ console.log(`[Electron Build] Using mocks: ${useMocks}`);
 // must be builded and not included in electron node_modules, because they are in TS.
 // Normal src/ folder is fine, because it's builded by webpack.
 const dependencies = Object.keys(pkg.dependencies).filter(
-    name => !(name.startsWith('@suite-common/') || name === '@trezor/utils'),
+    name =>
+        !(name.startsWith('@suite-common/') || ['@trezor/urls', '@trezor/utils'].includes(name)),
```

Comment:
Can you please make array in variable? Something named `buildedDependencies = [...]` it will be little bit more clear what it do

Reply:
Changed, squashed, rebased